com.dynamo.cr/com.dynamo.cr.bob.test/src/com/dynamo/bob/pipeline/LuaBuilderTest.java

```diff
@@ -58,7 +58,8 @@ public void testProps() throws Exception {
         src.append("go.property(\"material\", resource.material(\"/material.material\"))\n");
         src.append("\n");
         src.append("    go.property(  \"space_number\"  ,  1   )\n");
-        src.append("go.property(\"semi_colon\", 1); \n");
+        // we need something in-front of the go.property to "catch" the stray semi-colon when it has been removed
```

Comment:
Interesting thing is that the previous version of the LuaScanner actually removed the go.property() + 1 additional character. Now that it doesn't it resulted in a line with only a semi-colon on it, which isn't valid Lua causing the Lua builder to fail.

Reply:
but shouldn't it be fixed?
what if a user has such line in his code?
```
go.property("my", hash("something");
```

that means it will fail?

Reply:
Hmm, you're right. I suppose it is actually ok to consume the next character as well. It has to be either a space, comma, semi-colon or new line. I'll have to look closer at the code for removing a token.